05-Jupyter/05.1-ipython.md

```diff
@@ -0,0 +1,25 @@
+# IPython
+
+### TODO:
+- Read about IPython.
+- Explain the cons and pros of using Ipython over a standard Python shell.
+- Start an Ipython shell:
+    - `%time`
+    - `%debug`
+    - `%pdoc`
+    - `%%bash`
+    - `?` operator
+    - `??` operator
```

Comment:
What should I as a trainee do with these operators?

Reply:
%%bash is very useful, but it's mostly so they know these kind of commands exist